src/utilities/new_versions.jl

```diff
@@ -110,15 +114,17 @@ function create_new_pull_request(
     title::AbstractString,
     body::AbstractString,
 )
+    repo_owner = String(rsplit(repo.path_with_namespace, "/"; limit=2)[1])
```

Comment:
Should this be a helper that takes ` repo` ?

Reply:
Possibly? I'm on the fence about factoring out a single line that happens only once, as I don't think it's going to be used anywhere else.

Reply:
I think if we're doing this once it's fine. If we end up needing similar code multiple times we should then turn it into a helper function.